.github/workflows/release_version.yml

```diff
@@ -83,27 +76,44 @@ jobs:
           else
             export REL_VER=`git describe --tags --always`
           fi
+          if [[ $REL_VER == tikv-br* ]];then
+            export PORD_NAME=tikv-br
+            export PORD_DIR=br
+          elif [[ $REL_VER == tikv-cdc* ]];then
+            export PORD_NAME=tikv-cdc
+            export PORD_DIR=cdc
+          elif [[ $REL_VER == gc-worker* ]];then
+            export PORD_NAME=gc-worker
+            export PORD_DIR=gc-worker
+          fi
+
           echo ::set-output name=REL_VER::$REL_VER
-          make release;
+          echo ::set-output name=PORD_NAME::$PORD_NAME
+          echo ::set-output name=PORD_DIR::$PORD_DIR
+          cd $PORD_DIR
+          make
```

Comment:
Maybe better to be `make release`. Which has certain meaning, to avoid be unexpectedly modified of packages' Makefile.

Reply:
ok, then `cdc` need add the compile command `make relase` in Makefile.

Reply:
I think it worths to add the command.

Reply:
done